client/ayon_core/tools/publisher/widgets/overview_widget.py

```diff
@@ -293,6 +299,10 @@ def _on_product_change(self, *_args):
             instances, context_selected, convertor_identifiers
         )
 
+    def _on_double_clicked(self):
+        from ayon_core.tools.utils import host_tools
+        host_tools.show_publisher(tab="publish")
```

Comment:
Please do not do it this way. The publisher tool does not have to shown using `host_tools`. Resubmit signal to `PublisherWindow`.

Reply:
I am now emitting a publish_tab_requested signal.